src/NuGet.Indexing/Extraction/CatalogPackageMetadataExtraction.cs

```diff
@@ -238,18 +239,14 @@ private void AddSupportedFrameworks()
                         .GetSupportedFrameworks()
                         .ToList();
                 }
-                catch (ArgumentException ex) when (ex.Message.ToLowerInvariant().StartsWith("invalid portable"))
+                catch (Exception ex) when (
+                    (ex is ArgumentException && ex.Message.ToLowerInvariant().StartsWith("invalid portable"))
+                    || ex is FrameworkException
+                    || ex is PackagingException)
```

Comment:
The change here is that we now ignore `PackagingException`s, correct?

Reply:
Yes